app/models/zendesk/zendesk_ticket.rb

```diff
@@ -50,6 +50,8 @@ def base_comment_snippets
         [
           Zendesk::LabelledSnippet.new(on: self, field: :needed_by_date, label: "Needed by date"),
           Zendesk::LabelledSnippet.new(on: self, field: :not_before_date, label: "Not before date"),
+          Zendesk::LabelledSnippet.new(on: self, field: :needed_by_time, label: "Needed by time"),
+          Zendesk::LabelledSnippet.new(on: self, field: :not_before_time, label: "Not before time"),
```

Comment:
You may need to define two new methods in addition to this- see https://github.com/alphagov/support/blob/98f88ef5dba31485aae195939573ec8114d68866/app/models/zendesk/zendesk_ticket.rb#L20-L30